changelog/bug-1577839.md

```diff
@@ -0,0 +1,4 @@
+level: patch
+reference: bug 1577839
+---
+Including TagSpecifications in a launchConfig for the AWS provider now applies those tags to the indicated resources.  Before this version, those tags would be ignored.
```

Comment:
Can we re-phrase this as "Fixes the bug where TagSpecifications in a launchConfig for the AWS provider would be ignored etc" or something like that, because it wasn't deliberate or me not realizing this needed to be done - my intention _was_ to apply those tags, it's just that because of so many `launchConfigs` I didn't see I was making a mistake